tests/core/test_api.py

```diff
@@ -49,11 +51,23 @@ def test_generate_resource_urls_with_id(test_config: FidesctlConfig) -> None:
 
 # Integration Tests
 @pytest.mark.integration
-def test_api_ping(test_config: FidesctlConfig) -> None:
-    assert (
-        _api.ping(test_config.cli.server_url + API_PREFIX + "/health").status_code
-        == 200
-    )
+@pytest.mark.parametrize(
+    "database_health, expected_status_code",
+    [("healthy", 200), ("needs migration", 200), ("unhealthy", 503)],
+)
+def test_api_ping(
+    test_config: FidesctlConfig,
+    database_health: str,
+    expected_status_code: int,
+    monkeypatch: MonkeyPatch,
+) -> None:
+    def mock_get_db_health() -> str:
+        return database_health
+
+    # not working :(
```

Comment:
I'm trying to think of how we could do this without mocking, but it might not be possible....

i'll play around with this!